tests/logictest/suites/base/09_fuse_engine/09_0008_fuse_optimize_table

```diff
@@ -45,6 +45,16 @@ select segment_count,block_count from fuse_snapshot('db_09_0008', 't') limit 1;
 ----
 1 1
 
+statement ok
+optimize table `t` statistic;
+
+statement query II
+select segment_count,block_count from fuse_snapshot('db_09_0008', 't') limit 1;
```

Comment:
This sql will check what? It seems not related to statistic

Reply:
This will work:
1. check the statistic
2. optimize table `t` statistic;
3. check the statistic again, make the step2 works